compiler/src/dotty/tools/dotc/interpreter/Interpreter.scala

```diff
@@ -111,10 +111,13 @@ class Interpreter(implicit ctx: Context) {
         val env2 = bindings.foldLeft(env)((acc, x) => interpretStat(x, acc))
         interpretTreeImpl(expansion, env2)
 
+      case Typed(expr, _) =>
+        interpretTreeImpl(expr, env)
```

Comment:
Do we need a check here? Maybe not.

Reply:
What kind of check? This is a fully typed tree, we evaluate it's erased version.

Reply:
I was thinking to check the type, but it's not the semantics of the language. So ignore the comment.